homeassistant/components/tailwind/config_flow.py

```diff
@@ -182,6 +183,15 @@ async def async_step_reauth_confirm(
             errors=errors,
         )
 
+    async def async_step_dhcp(self, discovery_info: DhcpServiceInfo) -> FlowResult:
+        """Handle dhcp discovery to update existing entries.
+
+        This flow is triggered only by DHCP discovery of known devices.
+        """
+        await self.async_set_unique_id(format_mac(discovery_info.macaddress))
+        self._abort_if_unique_id_configured(updates={CONF_HOST: discovery_info.ip})
+        return self.async_abort(reason="already_configured")
+
```

Comment:
```suggestion
        return self.async_abort(reason="unknown_device")
```

If a config entry with the given unique id exists `self._abort_if_unique_id_configured` will raise an error.
If we come here, it means that it's a new device, and as we don't support setting up via DHCP we should raise a different error.

Please add the error also to strings.json

Reply:
I'm fine with making that change you suggest, however, that means more integrations needs updating, as this is how it was handled in other places too.

Reply:
Changed it to "unkown", as the situation should never happen.

Added extra code comments for clarity.